core/templates/pages/classroom-admin-page/modals/update-classrooms-order-modal.component.html

```diff
@@ -5,8 +5,7 @@ <h3>Update Classroom order</h3>
     <div *ngFor="let classroom of classroomIdToClassroomNameIndex"
          class="classroom-tile"
          cdkDrag
```

Comment:
Why are cdkDragStarted and cdkDragEnded removed and cdkDrag remains as is in the div element?

Reply:
Earlier we were adding custom classes to target the dragged tile and make it's size bigger but we can simply target cdk-drag-placeholder class. that's we removed cdkDragEnded and cdkDragStarted. they were used to add or remove class.

and we need cdkDrag directive to use the drag function.